ribs/visualize.py

```diff
@@ -103,7 +104,8 @@ def grid_archive_heatmap(archive,
         cmap (str, list, matplotlib.colors.Colormap): Colormap to use when
             plotting intensity. Either the name of a colormap, a list of RGB or
             RGBA colors (i.e. an Nx3 or Nx4 array), or a colormap object.
-        square (bool): If True, set the axes aspect ratio to be "equal".
+        aspect ('auto', 'equal', float) [optional]: the aspect ratio of the heatmap. Defaults to 'auto' for 2D and 0.3 for 1D. 'equal' is the same as `aspect=1`.
```

Comment:
Use double ticks for code stuff e.g. ` ``aspect=1`` `. Docstrings are formatted in restructured text, not Markdown.